### PR TITLE
[13.0][MIG] project_timeline_task_dependency: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
**Depends on** https://github.com/OCA/project/pull/656

depen

cc @tecnativa TT19800

### CODE REVIEW

File: project_timeline_task_dependency/README.rst
Comment (sergio-teruel) [inline project_timeline_task_dependency/README.rst]: Change web_timeline version to 13.xxxx

**Review by sergio-teruel — APPROVED** [review]
Code review

File: project_timeline_task_dependency/README.rst
Comment (pedrobaeza) [reply project_timeline_task_dependency/README.rst]: You can directly remove this instead, as there's no minimum version in this time (it was an exceptional case in 11.0)

Comment (sergio-teruel) [reply project_timeline_task_dependency/README.rst]: Ok...

**Comment by Tardo** [comment]
Changes Done!

**Comment by pedrobaeza** [comment]
Please rebase and check its correct behavior.

**Comment by Tardo** [comment]
Rebased and cleaned the 'bot' commits too.

Needs this fix to work: https://github.com/OCA/web/pull/1600

**Comment by pedrobaeza** [comment]
You should wait until the patch in OCA/web is merged and then relaunch for having proper runbot.

**Comment by pedrobaeza** [comment]
Please rebase again now that it's merged.

**Comment by Tardo** [comment]
Rebased

**Review by pedrobaeza — APPROVED** [review]
Tried on runbot and working

**Pull request merged**
